transactions/mint_nft.cdc

```diff
@@ -46,41 +44,34 @@ transaction(
 
     execute {
 
-        // Create the royalty details
-        var count = 0
-        var royalties: [MetadataViews.Royalty] = []
-        while royaltyBeneficiaries.length > count {
-            let beneficiary = royaltyBeneficiaries[count]
-            let beneficiaryCapability = getAccount(beneficiary)
-            .getCapability<&{FungibleToken.Receiver}>(MetadataViews.getRoyaltyReceiverPublicPath())
-
-            // Make sure the royalty capability is valid before minting the NFT
-            if !beneficiaryCapability.check() { panic("Beneficiary capability is not valid!") }
-
-            royalties.append(
-                MetadataViews.Royalty(
-                    receiver: beneficiaryCapability,
-                    cut: cuts[count],
-                    description: royaltyDescriptions[count]
-                )
-            )
-            count = count + 1
-        }
-
+        // // Create the royalty details
```

Comment:
Should we add a comment on how to handle royalties in NFTv2?
Commented code might suggest its not supported any more

Reply:
I commented it out because I was testing other things and forgot to uncomment it. I'll add something back for royalties